app/views/questionnaire.py

```diff
@@ -216,18 +215,19 @@ def get_timeout_continue(eq_id, form_type, collection_id):  # pylint: disable=un
     return 'true'
 
 
-@questionnaire_blueprint.route('session-expired', methods=["GET"])
+@questionnaire_blueprint.route('session-expired', methods=["POST"])
 @login_required
+def post_session_expired(eq_id, form_type, collection_id):  # pylint: disable=unused-argument
+    _remove_survey_session_data()
+    return get_session_expired(eq_id, form_type, collection_id)
+
+
+@questionnaire_blueprint.route('session-expired', methods=["GET"])
```

Comment:
ok, can you raise a debt card about knowing the theme from the URL rather than having to load the schema etc?